agent/plugin/plugin_test.go

```diff
@@ -55,25 +55,25 @@ func TestCreateFromJSON(t *testing.T) {
 		},
 		{
 			`["file://github.com/buildkite-plugins/docker-compose#a34fa34"]`,
-			[]*Plugin{&Plugin{
+			[]*Plugin{{
 				Location:      `github.com/buildkite-plugins/docker-compose`,
 				Version:       `a34fa34`,
 				Scheme:        `file`,
 				Configuration: map[string]interface{}{},
 			}},
 		},
 		{
-			`["github.com/buildkite-unofficial/ping#master"]`,
-			[]*Plugin{&Plugin{
-				Location:      `github.com/buildkite-unofficial/ping`,
+			`["github.com/buildkite-plugins/fake-plugin#master"]`,
```

Comment:
Does the agent have special behaviour for handling plugins in the `github.com/buildkite-plugins` org vs. plugins in third party GH orgs?

I assume this is OK because none of the tests are checking the special buildkite-plugins form  (`docker-compose#v3.7.0`), but I thought it was worth checking.

Reply:
Yeah I think that's the purpose of these tests — they should not be in `buildkite` or `buildkite-plugins`.

Reply:
ah good call! in that case, could someone with github admin superpowers create a buildkite org that's neither buildkite nor buildkite-plugins, and then we could use that?

Reply:
Do we need to register another GitHub org for testing this?
Or paramerise some test things so that we can use an existing one that we do already control?

Reply:
> Does the agent have special behaviour for handling plugins in the github.com/buildkite-plugins org vs. plugins in third party GH orgs?

TIL this happens server-side, not agent-side.

```ruby
    case paths.length
    when 1
      "github.com/buildkite-plugins/#{add_plugin_suffix(paths.first, uri.fragment)}"
    when 2
      "github.com/#{paths.first}/#{add_plugin_suffix(paths.last, uri.fragment)}"
    else
      source
    end
```

Reply:
(So, I think it's fine that we use `github.com/buildkite-plugins` for testing, unless we're really worried about future-proofing it against future agent-side handling of this)

Reply:
Good idea! Maybe you could use this demo org for now?

https://github.com/hashikite